src/views/networkpolicies/new/components/NetworkPolicyFormSDNAlert.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC, useState } from 'react';
+
+import { useFlag } from '@openshift-console/dynamic-plugin-sdk';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import ExternalLink from '@utils/components/ExternalLink/ExternalLink';
+import { FLAGS } from '@utils/constants';
+import { getNetworkPolicyDocURL, isManaged } from '@utils/constants/documentation';
+import { ClusterNetworkFeatures } from '@utils/hooks/useClusterNetworkFeatures';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+
+type NetworkPolicyFormSDNAlertProps = {
+  networkFeatures: ClusterNetworkFeatures;
+  networkFeaturesLoaded: boolean;
+};
+
+const NetworkPolicyFormSDNAlert: FC<NetworkPolicyFormSDNAlertProps> = ({
+  networkFeatures,
+  networkFeaturesLoaded,
+}) => {
+  const { t } = useNetworkingTranslation();
+  const [showSDNAlert, setShowSDNAlert] = useState(true);
+  const isOpenShift = useFlag(FLAGS.OPENSHIFT);
+
+  if (
+    !showSDNAlert ||
+    !networkFeaturesLoaded ||
+    networkFeatures?.PolicyEgress !== undefined ||
+    networkFeatures?.PolicyPeerIPBlockExceptions !== undefined
```

Comment:
I'd suggest to change this to isEmpty(...) in case it can be null or some other nullish value

Reply:
next pr